docs/add-package-into-cloudshell.md

```diff
@@ -0,0 +1,67 @@
+# Add a package into Cloud Shell
+
+One of the main features of Cloud Shell is that it serves many different packages to its users. Users don't have to go through the hassle of installing packages themselves when using Cloud Shell.
+
+## An installation work around
+
+For all packages that do not require root permissions, you are free to install it yourself. This includes, but not limited to, Python, Powershell, Nodejs packages.
+
+Here's an example as to how you can install a package like ORAS yourself:
+
+```
+cd ~
+VERSION="1.2.0"
+curl -LO "https://github.com/oras-project/oras/releases/download/v${VERSION}/oras_${VERSION}_linux_amd64.tar.gz"
+mkdir -p ~/oras-install/
+mkdir -p ~/.local/bin/
+tar -zxf oras_${VERSION}_*.tar.gz -C oras-install/
+mv ~/oras-install/oras ~/.local/bin/oras
+rm -rf oras_${VERSION}_*.tar.gz oras-install/
+```
+Reference: https://oras.land/docs/installation#linux
+
+For persistence of your packages across multiple Cloud Shell sessions, it is recommended you enable the persistent storage with the "storage account" feature of Cloud Shell. In addition, install these packages in any directory in `$HOME`, since the `$HOME` directory is stored in the storage account.
+
+In our above example you can see that the package is installed in the `~/.local/bin` directory. Since it is within a directory within `$HOME`, the ORAS package will persistent across Cloud Shell sessions (assuming you are using a storage mount).
+
+## Requesting the package
+
+As per security & compliance requirements, we are striving to have all our packages installed from Azure Linux (a.k.a Mariner). Azure Linux is a compliant Microsoft Linux Operating System. If you want your package to be available in Cloud Shell, we expect to have it downloadable from Azure Linux OS. 
+
```

Comment:
Add a link to docs/package-inclusion-guide.md which provides more detailed information about what packages might be included.